orbs/helm/circle-wait-job

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env ruby
+
+require("optparse")
+require("net/http")
+require("json")
+
+class CLI
+  attr_reader(:token, :organization, :project, :job, :branch, :interval, :vcs, :build_number)
+
+  def initialize
+    @branch = ENV["CIRCLE_BRANCH"]
+    @interval = 60
+    @vcs = "github"
+    @build_number = ENV["CIRCLE_BUILD_NUM"]
+    @token = ENV["CIRCLE_TOKEN"]
+    @job = ENV["CIRCLE_JOB"]
+    @organization = ENV["CIRCLE_PROJECT_USERNAME"]
+    @project = ENV["CIRCLE_PROJECT_REPONAME"]
+  end
+
+  def parse(args)
+    OptionParser.new do |parser|
+      define_options!(parser)
+      parser.parse!(args)
+    end
+    self
+  end
+
+  private
+
+  def define_options!(parser)
+    parser.banner = "Usage: circleci-wait-job OPTIONS"
+    parser.separator("")
+    parser.separator("OPTIONS")
+    parser.separator("")
+
+    parser.on("-t=VALUE", "--token=VALUE", "the circleci api token") { |value| @token = value }
+    parser.on("-o=VALUE", "--organization=VALUE", "the circleci organization name") { |value| @organization = value }
+    parser.on("-p=VALUE", "--project=VALUE", "the circleci project name") { |value| @project = value }
+    parser.on("-j=VALUE", "--job=VALUE", "the circleci job name") { |value| @job = value }
+    parser.on("-b=VALUE", "--branch=VALUE", "the circleci branch") { |value| @branch = value }
+    parser.on("-i=VALUE", "--interval=VALUE", "the wait time between pooling") { |value| @interval = value }
+    parser.on("--vcs=VALUE", "the version control system (default: github)") { |value| @vcs = value }
+    parser.on("--current-build-number=VALUE", "the current circleci build number") { |value| @build_number = value }
+
```

Comment:
You should probably check that you got a non-empty value for some mandatory settings, since the default value comes from the environment.